build.gradle

```diff
@@ -86,11 +86,14 @@ jacocoTestReport {
 }
 
 //NOTE: we ignore contracts for now
+// TEMPORARY HACK: JP removed -Werror because
```

Comment:
I know, sorry, fixed that since but haven't pushed yet because I'm moving code around.